### ISSUE TITLE
Changer d'éditeur perd le reste du contenu du formulaire

### ISSUE THREAD

**Issue by viki53** [zestedesavoir/zds-site#5811]
**Description du bug**

En revenant à l'éditeur classique le contenu du texte édité est bien conservé, mais le reste du formulaire est perdu.

**Comment reproduire ?**

1. Créer un nouveau tutoriel
2. Entrer un titre, une description, etc.
3. Changer d'éditeur pour l'intoduction

**Comportement attendu**

Le contenu de l'ensemble du formulaire devrait être conservé, pas juste le champ pour lequel on change d'éditeur

**Informations techniques**

- OS: macOS
- Navigateur: Firefox 76.0.1
- Version/environnement de zds: production (29.1)

**Comment by A-312** [comment]
Similaire à : #5753

**Comment by viki53** [comment]
Effectivement, je l'avais pas vu en cherchant un éventuel ticket existant.

**Comment by A-312** [comment]
(j'ai eu du mal à le retrouver aussi 😭 )
